### ISSUE TITLE
Videos list of a channel is empty

### ISSUE THREAD

**Issue by lelapin123** [TeamNewPipe/NewPipe#9281]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

0.24.0

### Steps to reproduce the bug

How to reproduce the bug:

1. Go to a video and click on the channel title (for example 52Kards)
2. You just see the title of the channel and not the list of videos

How to NOT reproduce the bug:

1. Go to a video and click on the channel title (for example "Taylor Swift" or "Justin Bieber")
2. You see the title of the channel and the list of videos

I suspect an update in progress.
Bottom line: when i try to update the subscription feed, I don't have anymore update since 5 days ago...

### Expected behavior

I expect to see the list of videos of a channel when i click on the channel title (previous behaviour)

### Actual behavior

It stopped to work

### Affected Android/Custom ROM version

Android 12

### Affected device model

Huawei p30 pro

**Comment by NoPlagiarism** [comment]
Duplicate of #9223
Stop it. And close it.

**Comment by lelapin123** [comment]
Duplicate
